core/arch/arm/plat-zynqmp/conf.mk

```diff
@@ -27,3 +27,23 @@ CFG_SHMEM_SIZE   ?= 0x10000000
 
 CFG_WITH_STATS ?= y
 CFG_CRYPTO_WITH_CE ?= y
+
+CFG_ZYNQMP_PM ?= $(CFG_ARM64_core)
+
+ifeq ($(CFG_RPMG_FS),y)
```

Comment:
Typo, should be CFG_RPMB_FS instead (not G).

Reply:
yeah, that is the risk of doing forward ports... thanks!